sources/web/datalab/static/dark.css

```diff
@@ -113,6 +113,16 @@ div.output {
 div.output_stderr {
   background-color: #a94442;
 }
+input.raw_input {
+  color: #ddd;
+  background-color: #333;
+}
+input.raw_input::selection {
+  background: #ccc;
+}
+input.raw_input::-moz-selection {
+  background: #999;
```

Comment:
> It would actually be great if we can merge the two CSS rules into one that takes both selectors. 

Unfortunately, the following didn't work in my testing.

```
input.raw_input::selection, input.raw_input::-moz-selection {
  background: rgba(85, 85, 85, 0.99);
}
```
The following snippet from [Mozilla Developer Network](https://developer.mozilla.org/en-US/docs/Web/CSS/::selection) also reaffirms the test result:

> Due to the fact that the CSS parsing rules require dropping the whole rule when encountering an invalid pseudo-element, two separate rules must be written..